cypress.json

```
@@ -0,0 +1,4 @@
+{
+  "baseUrl": "https://app.neetoauth.com",
```

We always use staging url for testing.
```suggestion
  "baseUrl": "https://app.neetoauth.net",
```